test/integration/test_model_output_integration.py

```diff
@@ -41,7 +40,7 @@ def expected_model_output_schema() -> pa.Schema:
     return expected_schema
 
 
-def test_missing_model_output_id_numeric(tmpdir, test_file_path):
+def test_missing_model_output_id_numeric(tmpdir, test_file_path, schema_origin_date):
```

Comment:
Another quick question just to confirm what I'm understanding about python mocking. So the `schema_origin_date` fixture is an object that is passed to a test, effectively triggering the masking/mocking of `hubverse_transform.model_output.ModelOutputHandler._get_schema` when called and returning the  schema defined in `schema_origin_date`, correct?

Reply:
Close. (I'm new-ish to pytest library, BTW. It's different from Python's built-in one, [unittest](https://docs.python.org/3/library/unittest.html) - [differences discussed here](https://www.reddit.com/r/Python/comments/18bjv0y/pytest_over_unittest/)) `schema_origin_date` is a [pytest fixture](https://docs.pytest.org/en/stable/explanation/fixtures.html), that is, a function that's called before the test. Here it mocks the two internal methods that need to be changed to bypass actually loading a tasks.json (`_read_tasks()`) and getting a corresponding schema `_get_schema()`). The mocking is done via https://pypi.org/project/pytest-mock/ , which itself provides the `mocker` fixture that `schema_origin_date` uses. HTH!